### PR TITLE
Cache ord-editor layers in tests

### PR DESCRIPTION
**Pull request by skearnes**
With 100% caching (no changes to any layers), the test time drops from 7:24 at HEAD to 2:51 in this PR.

### CODE REVIEW

**Comment by skearnes** [comment]
Hold off on reviewing; the subsequent editor tests aren't using the cache like I expected.

**Comment by skearnes** [comment]
> Hold off on reviewing; the subsequent editor tests aren't using the cache like I expected.

PTAL; with 100% caching (no changes to any layers), the test time drops from 7:24 at HEAD to 2:51 in this PR.

File: .github/workflows/run_tests.yml [hunk 1]
```diff
@@ -49,3 +42,9 @@ jobs:
         npm install puppeteer
         export ORD_EDITOR_POSTGRES_PASSWORD=test
         ./run_tests.sh "${CACHE_TARGET}"
+    - name: Update docker cache
+      run: |
+        cd "${GITHUB_WORKSPACE}"
+        docker tag openreactiondatabase/ord-editor "${CACHE_TARGET}" && docker push "${CACHE_TARGET}" || true
```
Comment (antonkast-google) [inline .github/workflows/run_tests.yml]: "|| true"?

Comment (skearnes) [reply .github/workflows/run_tests.yml]: The test shouldn't fail if the cache can't be updated for some reason. But maybe we want the failure so we can spot caching errors. I'll remove.

**Review by antonkast-google — APPROVED** [review]

**Pull request merged**